lib/say_cases.rb

```diff
@@ -0,0 +1,46 @@
+class SayCase < OpenStruct
+  def test_name
+    'test_%s' % description.gsub(/[-,]/,'_').downcase.tr(' ', '_')
```

Comment:
This is now failing CI checks due to us updating the Rubocop style checks.
There needs to be a space after the comma in this line.